resource_customizations/numaflow.numaproj.io/InterStepBufferService/actions/discovery.lua

```diff
@@ -0,0 +1,18 @@
+local actions = {}
+actions["force-promote"] = {
+  ["disabled"] = true,
+  ["iconClass"] = "fa-solid fa-fw fa-forward"
+}
+
+-- force-promote
+local forcePromote = false
+if (obj.metadata.ownerReferences ~= nil and obj.metadata.ownerReferences[1] ~= nil and obj.metadata.ownerReferences[0].kind == "ISBServiceRollout") and (obj.metadata.labels ~= nil and obj.metadata.labels["numaplane.numaproj.io/upgrade-state"] == "in-progress") then
```

Comment:
should this say `ownerReferences[0]` instead of `ownerReferences[1]`? 

You may even be able to just simplify this to only include "numaplane.numaproj.io/upgrade-state" label

Reply:
Went ahead and removed it.